tests/perf/test_classification.py

```diff
@@ -0,0 +1,179 @@
+"""OTX Classification perfomance tests."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+import pytest
+
+from otx.cli.registry import Registry
+from .benchmark import OTXBenchmark
+
+
+MODEL_TEMPLATES = Registry(f"src/otx/algorithms").filter(task_type="CLASSIFICATION").templates
+MODEL_IDS = [template.model_template_id for template in MODEL_TEMPLATES]
+
+
+class TestPerfSingleLabelClassification:
+    """Benchmark single-label classification."""
+
+    BENCHMARK_CONFIGS = {
+        "small": {
+            "tags": {
+                "task": "single_label_classification",
+            },
+            "datasets": [
+                "classification/single_label/multiclass_CUB_small/1",
+                "classification/single_label/multiclass_CUB_small/2",
+                "classification/single_label/multiclass_CUB_small/3",
+            ],
+            "num_repeat": 3,
```

Comment:
It seems that small dataset is sampled three times. If then, it's executed total 9 times because "num_repeat" is set as 3. Is it intended?

Reply:
Yes, it's for benchmark stability. small x 3 x 3 / medium x 3 / large x 1 is default.